mlir/lib/Transforms/ViewOpGraph.cpp

```diff
@@ -276,6 +277,8 @@ class PrintOpPass : public impl::ViewOpGraphBase<PrintOpPass> {
   /// Process a block. Emit a cluster and one node per block argument and
   /// operation inside the cluster.
   void processBlock(Block &block) {
+    sortTopologically(&block);
```

Comment:
Is the printer not modifying the code and this would change this invariant? This can be undesirable in the context of calling this as debugging helper at various stages of a pass pipeline

Reply:
Good point, I didn't consider that this pass can be used in the middle of the pipeline. I've updated the code so that it does not modify the existing block.